engine/verification/fetcher/engine.go

```diff
@@ -249,7 +258,51 @@ func (e *Engine) validateChunkDataPack(chunk *flow.Chunk, senderID flow.Identifi
 	}
 
 	// 3. collection id must match
+	err := e.validateCollectionID(collection, chunkDataPack, chunk.Index, result)
+	if err != nil {
+		return err
```

Comment:
better to inlucde `senderID`, blockID, chunkID, resultID, for debugging purpose. same as L252

Reply:
fixed in https://github.com/onflow/flow-go/pull/686/commits/766b28bcbb82fc8736161c4378212a69080801dd, but all errors returning from this method are logged [here](https://github.com/onflow/flow-go/blob/766b28bcbb82fc8736161c4378212a69080801dd/engine/verification/fetcher/engine.go#L182-L187) which includes all this info again. So, the applied fix is a kind of redundancy.